layouts/partials/docs/content-page.html

```diff
@@ -5,8 +5,7 @@
     Edit This Page
   </a>
 </p>
-{{ if not .page.Params.notitle }}
```

Comment:
hey ... the `notitle` param is still relevant in some cases, right?

Reply:
👀

Reply:
I had searched for `notitle` in the site, but perhaps I missed something.
Just added this check back.